### PR TITLE
release notes for 1.16

### PR DESCRIPTION
**Pull request by fmcquillan99**
Release notes from Apache JIRAs

### CODE REVIEW

File: RELEASE_NOTES [hunk 1]
```diff
@@ -8,10 +8,47 @@ A complete list of changes for each release can be obtained by viewing the git
...
+Bug fixes:
+    - madlib.dist_jaccard not releasing memory (MADLIB-1291)
+    - MLP with minibatching fails on postgres (MADLIB-1302)
+    - MLP does not stop even after tolerance reached (MADLIB-1325)
+    - MLP warm start not working (MADLIB-1329)
+    - Pivot: Fix array_agg + distinct scaling issue on gpdb (MADLIB-1361)
+    - linregr_train fails when dependent variable is a JSONB element (MADLIB-1284)
+    - MADLib 1.15 does not recognize Postgres 10 declarative partitioned table (MADLIB-1287)
+    - Encoding module is not handling bigint properly (MADLIB-1295)
+    - MLP with minibatch fails for integer dependent variable on PostgreSQL (MADLIB-1322)
```
Comment (orhankislal) [inline RELEASE_NOTES]: We usually group the items by module to make it easier to follow.

File: RELEASE_NOTES [hunk 2]
```diff
@@ -8,10 +8,47 @@ A complete list of changes for each release can be obtained by viewing the git
...
+New features:
+    - Deep learning: support for Keras with TensorFlow backend with GPU acceleration (MADLIB-1268, MADLIB-1304, MADLIB-1305, MADLIB-1307, MADLIB-1308, MADLIB-1309, MADLIB-1310, MADLIB-1311, MADLIB-1313, MADLIB-1314, MADLIB-1315, MADLIB-1316, MADLIB-1319, MADLIB-1321, MADLIB-1326, MADLIB-1330, MADLIB-1335, MADLIB-1336, MADLIB-1338, MADLIB-1343, MADLIB-1348, MADLIB-1349, MADLIB-1350, MADLIB-1356, MADLIB-1357, MADLIB-1358, MADLIB-1324, MADLIB-1337, MADLIB-1347, MADLIB-1363)
+    - Deep learning: utility to load model architectures and weights (MADLIB-1306)
+    - Deep learning: preprocess images for gradient descent optimization algorithms (MADLIB-1290, MADLIB-1332, MADLIB-1334, MADLIB-1300, MADLIB-1303)
+    - kd-tree method for k-nearest neighbors for faster approximate solution (MADLIB-1061, MADLIB-1293)
+    - Support for Greenplum 6 (MADLIB-1298)
+    - Support for PostgreSQL 11 (MADLIB-1283)
+
+Bug fixes:
+    - madlib.dist_jaccard not releasing memory (MADLIB-1291)
```
Comment (orhankislal) [inline RELEASE_NOTES]: No need for `madlib.`

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/986/

File: RELEASE_NOTES (hunk 1 above)
Comment (fmcquillan99) [reply RELEASE_NOTES]: OK, will reorder

File: RELEASE_NOTES (hunk 2 above)
Comment (fmcquillan99) [reply RELEASE_NOTES]: removed `madlib`

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/987/

**Pull request closed without merging**